.github/workflows/linuxbuildlib

```diff
@@ -19,28 +19,22 @@ set -evu
 # Choose a Python to install conan
 export PYBIN=/opt/python/cp39-cp39/bin
 
-export CONAN_REVISIONS_ENABLED=1
-
 ${PYBIN}/pip install --upgrade pip
-${PYBIN}/pip install conan~=1.53
+${PYBIN}/pip install conan~=2.0.2
```

Comment:
```suggestion
${PYBIN}/pip install conan==2.0.2

Reply:
See question below, do we want this version or 2.0.x ?

Reply:
Happy to have compatibility (`~=`) here, but maybe actually it should be `~=2.0` ... will change.

Reply:
Changed.